src/main/java/mops/businesslogic/file/FileServiceImpl.java

```diff
@@ -158,12 +167,12 @@ public Directory deleteFile(Account account, long fileId) throws MopsException {
 
         try {
             fileInfoService.deleteFileInfo(fileId);
-            fileRepository.deleteFile(fileId);
+            deleteFile(fileId);
```

Comment:
```suggestion
            fileRepositorydeleteFile(fileId);
```
Unnecessary additional invoke and disturbs readability.